e2e/cypress/integration/local/project.spec.ts

```diff
@@ -133,14 +136,15 @@ describe("display migration information for anon user", () => {
 
 describe("display migration information for observer user", () => {
   const fixtures = new Fixtures(cy);
+  const params = "git_url=https%3A%2F%2Fdev.renku.ch%2Fgitlab%2Fe2e%2Flocal-test-project.git&branch=master";
   beforeEach(() => {
     fixtures.config().versions().userTest();
     fixtures.projects().landingUserProjects().projectTestObserver();
     cy.visit("/projects/e2e/local-test-project");
   });
 
   it("displays up-to-date migration", () => {
-    fixtures.projectMigrationUpToDate();
+    fixtures.projectMigrationUpToDate(params);
```

Comment:
With the changes suggested to `fixtures/projects.ts`, this can be changed back to:
```suggestion
    fixtures.projectMigrationUpToDate();
```